### DISCUSSION TITLE
ESP8266 is possible ?

### DISCUSSION THREAD

**Discussion by djgel** [Yogui79/IntexPureSpa#12] [Q&A] [answered]
Hi, I tried it with the ESP32 that I have here and everything is fine! But I also have ESP8266 boards that are easier to find and at a lower cost, is it possible tb to use this solution by changing the source code? Or are you limited to these boards due to the need for integration and libraries?

Tks

**Comment by Yogui79** [comment, marked as answer]
Hy,
Honestly I haven't tested it with an esp8266 because I had a bad experience with software serial on a esp8266.  I used an esp32 because he have more than one  on chip serial connection
